docs/source/quantization/fp8_e4m3_kvcache.rst

```diff
@@ -0,0 +1,38 @@
+.. _fp8_e4m3_kvcache:
+
+FP8 E4M3 KV Cache
+==================
+
+Quantizing the KV cache to FP8 reduces its memory footprint. This increases the number of tokens that can be stored in the 
+cache, improving throughput. OCP specifies two common floating point data formats: E5M2 (5 exponent bits and 2 mantissa 
```

Comment:
What is OCP? explain in this doc for users.

Reply:
solved